src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState: {
+    value: [],
+  },
+  reducers: {
+    add: (state, action) => {
+      const { title } = action.payload;
+      state.value.push({ title });
+    },
```

Comment:
Your reducer is missing the author and the id values. Kindly add them here so that it will be easier for you to implement it into the components in the future. You can consider doing something like the below code: 


```javascript
 add: (state, action) => {
      state.value = [
        ...state.value,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
```

Reply:
Thank you so much for your review, I have made the required adjustments.